index.html

```diff
@@ -6287,18 +6291,37 @@ <h4><dfn>Element Send Keys</dfn></h4>
    </dd>
 
    <dt><var><a>element</a></var> is <a>content editable</a>
-    <dd>If <var><a>element</a></var> does not currently have focus,
-      set the text insertion caret after any child content.
+    <dd>
+      <ol>
+        <li><p>If <var>element</var> is not the <a>active element</a>:
+          <ol>
+           <li><p>Run the <a>focusing steps</a> for <var>element</var>.
+
+            <li><p>Let <var>child count</var> be the [=map/size=]
+            of <var>element</var>'s [=tree/children=].
+
+            <li><p>Let <var>range</var> be a <a>live range</a> with [=range/start=]
+            and [=range/end=] (<var>element</var>, <var>child count</var>).
+
+            <li><p>Set the <a>active document</a>'s <a>selection</a>
+            to <var>range</var>.
```

Comment:
Does it mean to select the whole element's content?

Reply:
It's supposed to be a collapsed range, whose start and end node is `element` and whose offset is `child count` i.e. a zero-length selection corresponding to a caret position at the end of the element.